index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>freeCodeCamp Survey Form</title>
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
- [ ] I kindly suggest you fix this link by making the href value  exactly same as the name of your css file. <link rel="stylesheet" href="style.css" />. Css file will not be properly linked to html file if there are spelling errors in the link in your <head> tag.